distribution/istio-resources/kustomization.yaml

```diff
@@ -2,11 +2,14 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- deny-all-authorizationpolicy.yaml
```

Comment:
This policy shouldn't be removed. @karlschriek Did you only perform testing with this policy removed?

Reply:
The diff here is a bit misleading. deny-all hasn't been removed, see the full file! 

https://github.com/argoflow/argoflow-aws/blob/498fcc56f8ac396bbc7d6feb31158323bf3c686c/distribution/istio-resources/kustomization.yaml

Reply:
Thanks, that was my main worry